common/djangoapps/terrain/stubs/youtube.py

```diff
@@ -93,6 +93,9 @@ def do_GET(self):
 
         elif 'get_youtube_api' in self.path:
             if self.server.config.get('youtube_api_blocked'):
+                delay = self.server.config.get('youtube_api_blocked_delay')
+                if delay is not None:
+                    time.sleep(delay)
```

Comment:
Can we just not move `time.sleep(self.server.config.get('time_to_response', self.DEFAULT_DELAY_SEC))` above line 95?

If not, I would prefer to call this `youtube_api_timeout_delay`.